docs/Tutorial.md

```diff
@@ -76,7 +76,7 @@ export default App;
 
 That's enough for react-admin to render an empty app and confirm that the setup is done: 
 
-![Empty Admin](./img/tutorial_empty.png)
+[![Empty Admin](./img/tutorial_empty.png)](https://marmelab.com/react-admin/img/tutorial_empty.png)
```

Comment:
Can you please use relative links instead? It helps testability with the `make doc` command

Reply:
All absolute links have been replaced with the relative links.